packages/web-react/src/components/FieldGroup/README.md

```diff
@@ -0,0 +1,124 @@
+    <!-- Form fields… -->
+</FieldGroup>
+```
+
+⚠️ Remember the `label` prop should be always set to provide an accessible label for the group.
+
+## Helper Text
+
+To render helper text, add the `helperText` prop:
+
+```javascript
+<FieldGroup helperText="Helper text" id="example-field-group" label="Label">
+    <!-- Form fields… -->
+</FieldGroup>
+```
+
+## Fluid Width
+
+To make the FieldGroup component fluid, add the `isFluid` prop:
+
+```javascript
+<FieldGroup id="example-field-group" label="Label" isFluid>
+    <!-- Form fields… -->
+</FieldGroup>
+```
+
+## Disabled State
+
+To render FieldGroup as required, add the `isDisabled` prop:
+
+```javascript
+<FieldGroup id="example-field-group" label="Label" isDisabled>
+    <!-- Form fields… -->
+</FieldGroup>
+```
+
+⚠️ Remember to also disable all fields in the group using the `isDisabled` prop.
+
+👉 Read more about the disabled state in the [`web` implementation][gh-web-field-group-disabled] of `FieldGroup`.
+
+## Validation States
+
+Just like any other form component in Spirit, FieldGroup implements the
+[Validation state dictionary][dictionary-validation].
+
+Validation states can be presented either by adding the `validationState` attribute.
+
+```javascript
+<FieldGroup
+  id="example-field-group"
+  label="Label"
+  validationState="success"
+  validationText="Success validation message"
+/>
+```
+
+## API
+
+| Prop name               | Type                                           | Default | Required | Description                                |
+| ----------------------- | ---------------------------------------------- | ------- | -------- | ------------------------------------------ |
+| `helperText`            | `string`                                       | `null`  | no\*\*   | Custom helper text                         |
+| `form`                  | `string`                                       | `null`  | no       | Parent form ID                             |
+| `id`                    | `string`                                       | —       | yes      | Group and label identification             |
+| `isDisabled`            | `bool`                                         | `false` | no       | If true, the group is disabled             |
+| `isLabelHidden`         | `bool`                                         | `false` | no       | If true, label is hidden                   |
+| `isRequired`            | `bool`                                         | `false` | no       | If true, the group is marked as required   |
+| `label`                 | `string`                                       | `null`  | no\*     | Label text                                 |
+| `name`                  | `string`                                       | `null`  | no       | Group name                                 |
+| `UNSAFE_helperText`     | `string`                                       | `null`  | no\*\*   | Unescaped custom helper text (allows HTML) |
```

Comment:
We do not have `UNSAFE_` props here. I guess this is a copy paste from **web-twig** ;-)

Reply:
removed `UNSAFE_helperText`, `UNSAFE_label` and `UNSAFE_validationText`